docker-compose.yml

```diff
@@ -2,7 +2,13 @@ version: '3.7'
 
 services:
     postgres:
-        image: postgres
+        image: postgres:11
+        environment:
+            - POSTGRES_DB=${POSTGRES_DB}
+            - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+            - POSTGRES_USER=${POSTGRES_USER}
+        ports:
+            - 5432:5432
```

Comment:
Why do you publish the port?
Will you connect to postgresql directly from your computer ?

Reply:
Indeed it's not necessary, I just deleted this